File: formManagement/src/main/java/com/quizGpt/formManagement/Account/Dto/LoginResponseDto.java
package com.quizGpt.formManagement.Account.Dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class LoginResponseDto {
    private String token;
    private String email;
    private int id;
    private List<String> roles;
    private String authScheme;
    private String username;
    private String password;
    private String statusCode;
    private int statusCodeValue;
}

File: resource-server/src/account/dtos/loginResponseDto.ts
export class LoginResponseDto {
  token: String;
  authScheme: String;
  id: number;
  username: String;
  email: String;
  roles: String[];
  statusCode: String;
  statusCodeValue: number;

  constructor(
    token: String,
    authScheme: String,
    id: number,
    username: String,
    email: String,
    roles: String[],
    statusCode: String,
    statusCodeValue: number
  ) {
    this.token = token;
    this.authScheme = authScheme;
    this.id = id;
    this.username = username;
    this.email = email;
    this.roles = roles;
    this.statusCode = statusCode;
    this.statusCodeValue = statusCodeValue;
  }
}
